relayer/workers/parachaincommitmentrelayer/parachain-commitment-listener.go

```diff
@@ -14,26 +14,27 @@ import (
 	"golang.org/x/sync/errgroup"
 
 	"github.com/snowfork/polkadot-ethereum/relayer/chain"
+	"github.com/snowfork/polkadot-ethereum/relayer/chain/parachain"
 	chainTypes "github.com/snowfork/polkadot-ethereum/relayer/substrate"
 )
 
-type Listener struct {
-	config   *Config
-	conn     *Connection
+type ParachainCommitmentListener struct {
+	config   *parachain.Config
+	conn     *parachain.Connection
 	messages chan<- []chain.Message
 	log      *logrus.Entry
 }
 
-func NewListener(config *Config, conn *Connection, messages chan<- []chain.Message, log *logrus.Entry) *Listener {
-	return &Listener{
+func NewParachainCommitmentListener(config *parachain.Config, conn *parachain.Connection, messages chan<- []chain.Message, log *logrus.Entry) *ParachainCommitmentListener {
```

Comment:
Lets break this method signature up along multiple lines for better readability.